package.json

```diff
@@ -22,6 +23,8 @@
     "bootstrap": "4.0.0-alpha.6",
     "compression": "^1.6.2",
     "express": "^4.14.0",
+    "husky": "^0.14.3",
```

Comment:
Maybe we should install them in devDependencies by  `yarn add lint-staged husky --dev`

Reply:
I often use air-bnb, I can adjust everything